Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -85,10 +86,8 @@ public void TestProtocExecution()
         [Fact]
         public void TestThatSomeCodeIsGenerated()
         {
-            // Test that the code generator executes, and produces *something*.
-            // At the moment the code produced is not formatted, which means all
-            // whitespace is missing, and is therefore unparsable as C# code.
-            // TODO: Remove this test.
+            // Test that the code generator executes, and produces the expected code.
+            // TODO: Improve this testing infrastructure.
```

Comment:
Yup, I think at this point we're getting to the stage where it would be good to have "expected output" to review. If you could parse that, normalize both the actual output and expected output, and compare them, that would be great.

Reply:
Completely agree. I'll work on the testing in a PR soon.
(At the moment, I think PR means "Pouring with Rain")